client/lib/polyfill/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import localStoragePolyfill from 'lib/local-storage';
+
+export default ( () => {
+	if ( 'undefined' === typeof window ) {
```

Comment:
Hrm, just realized I may not understand how modules evaluate as well as I thought. Does code in the body of the module run before the module is `require`d? Or just once on the first `require`?